site/assignments/country-catalogue/country-catalogue.rst

```diff
@@ -156,6 +156,15 @@ Part 13 --- Filter by Density
 Part 14 --- Most Populous Continent
 -----------------------------------
 
+Write a method ``most_populous_continent`` that returns the name of the continent (as a string) that has the largest
+population based on information within the ``CountryCatalogue``. For example, although Asia has a large population,
+if no ``Country`` objects from Asia were included in the ``CountryCatalogue``, then those populations are not considered
+in the calculation. If the the ``CountryCatalogue`` is empty, this method raises an ``IndexError`` exception. This
```

Comment:
_For example_ feels out of place here. There's no example, rather clarifying the question, framing the query in terms of the data in the catalogue rather than data of Earth.

Reply:
I had actually spent time thinking of the _for example_ before I submitted the PR. Nice to know I'm not crazy. 

Either way, did a big ol' re-ordering of the info to help.